src/main/groovy/nebula/plugin/dependencylock/DependencyLockTaskConfigurer.groovy

```diff
@@ -65,21 +61,14 @@ class DependencyLockTaskConfigurer {
         this.project = project
     }
 
-    String configureTasks(String globalLockFilename, DependencyLockExtension extension, DependencyLockCommitExtension commitExtension, Map overrides) {
-        String lockFilename = project.hasProperty(LOCK_FILE) ? project[LOCK_FILE] : null
-        File lockFileInBuildDir = new File(project.buildDir, lockFilename ?: extension.lockFile)
-        if (project.hasProperty(USE_GENERATED_LOCK)) {
-            lockFilename = lockFileInBuildDir.path
```

Comment:
This caused that `lockFileInProjectDir` was invalid since it created concatenated 2 absolute paths. I haven't found any usage internally so I decided we might get rid of the feature at all.